akka-actor-typed/src/main/scala/akka/actor/typed/delivery/ConsumerController.scala

```diff
@@ -58,7 +58,7 @@ object ConsumerController {
    * Initial message from the consumer actor. The `deliverTo` is typically constructed
    * as a message adapter to map the [[Delivery]] to the protocol of the consumer actor.
    *
-   * If the producer is restarted it should send a new `Start` message to the
```

Comment:
Producers do not interact directly with `ConsumerController`, it was unaware of the `ConsumerController` existence